src/main/java/ru/hogwarts/school/service/FacultyService.java

```diff
@@ -0,0 +1,43 @@
+package ru.hogwarts.school.service;
+
+import org.springframework.stereotype.Service;
+import ru.hogwarts.school.model.Faculty;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class FacultyService {
+
+    private final Map<Long, Faculty> faculties = new HashMap<>();
+    private long lastID = 0;
+
+    public Faculty createFaculty(Faculty faculty) {
+        faculty.setId(lastID++);
```

Comment:
лучше начинать с 1 значения, тогда надо использовать префиксную запись
```suggestion
        faculty.setId(++lastID);
```

Reply:
+